src/main/java/seedu/address/model/person/Salary.java

```diff
@@ -83,10 +83,10 @@ public double getNetSalary() {
     public String getNetSalaryString() {
         DecimalFormat df = new DecimalFormat("0.00");
         String basicSalary = "Basic Pay: $" + super.toString();
-        String deductions = "Total Deductions: $" + df.format(getTotalDeductions());
-        String benefits = "Total Benefits: $" + df.format(getTotalBenefits());
-        String netSalary = "Net Salary: $" + df.format(getNetSalary());
-        return netSalary + "\n" + basicSalary + "\n" + deductions + "\n" + benefits;
+        String deductions = "Total Deductions: $" + getTotalDeductions();
+        String benefits = "Total Benefits: $" + getTotalBenefits();
+        String netSalary = "Net Salary: $" + getNetSalary();
+        return String.format("%1$-40s %2$-40s\n%3$-40s %4$-40s", netSalary, basicSalary, deductions, benefits);
```

Comment:
Does removing the `df.format()` and adding `%1$-40s %2$-40s\n%3$-40s %4$-40s` ensure the payments are returned in 2dp? Because the `getNetSalaryString()` test is failing now.

Reply:
I think `df.format()` is still needed.
![WeChat17f9d74ffb11e9f7ec88b60584477b1a](https://github.com/AY2324S1-CS2103T-W12-2/tp/assets/122149501/b4db1718-5d25-45e4-9b5f-7b6e83adedcd)

Reply:
```java
    public String getNetSalaryString() {
        DecimalFormat df = new DecimalFormat("0.00");
        String basicSalary = "Basic Pay: $" + super.toString();
        String deductions = "Total Deductions: $" + df.format(getTotalDeductions());
        String benefits = "Total Benefits: $" + df.format(getTotalBenefits());
        String netSalary = "Net Salary: $" + df.format(getNetSalary());
        return String.format("%1$-40s %2$-40s\n%3$-40s %4$-40s", netSalary, basicSalary, deductions, benefits);
    }
```